.cirrus.yml

```diff
@@ -311,6 +315,9 @@ task:
         GCLOUD_SERVICE_ACCOUNT_KEY: ENCRYPTED[f12abe60f5045d619ef4c79b83dd1e0722a0b0b13dbea95fbe334e2db7fffbcd841a5a92da8824848b539a19afe0c9fb]
         SHARD: tests
         SUBSHARD: framework_other
+        GOLDCTL: "C:\\Windows\\Temp\\goldctl_tool\\goldctl.exe"
```

Comment:
Same here, does cirrus allow the use of env variables here to reference Temp?

Reply:
Hm, maybe never mind about `$env:TEMP`  here and in the powershell script. Looks like we are hardcoding C:\Windows\Temp everywhere....